kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/ApiUtils.scala

```diff
@@ -60,7 +60,8 @@ object ApiUtils extends Logging {
       opEvent.sessionUser,
       opEvent.sessionType,
       operation.getSession.asInstanceOf[KyuubiSession].connectionUrl,
-      operation.metrics.asJava)
+      operation.metrics.asJava,
+      opEvent.progress)
```

Comment:
How about we use `operation.getOperationJobProgress` here?

Reply:
addressed, PTAL